.gitmodules

```diff
@@ -26,6 +26,6 @@
 [submodule "node-red-contrib-agile-solid"]
 	path = node-red-contrib-agile-solid
 	url = https://github.com/Agile-IoT/node-red-contrib-agile-solid.git
-[submodule "node-red-contrib-thingspeak42"]
-	path = node-red-contrib-thingspeak42
-	url = https://github.com/clough42/node-red-contrib-thingspeak42.git
+[submodule "node-red-contrib-agile-thingspeak"]
```

Comment:
It is true! I have forgotten it. @Exulansis made some minor modifications, so I think it is right to keep the agile repo.